docs/css/main.css

```diff
@@ -0,0 +1,472 @@
+.container {
+  max-width: 1280px;
+  padding: 0 1.5rem;
+  margin: 0 auto; }
+
+.baga-icon-container {
+  height: 20px;
+  display: flex;
+  position: relative;
+  margin: 1.5rem 0;
+  z-index: 999; }
+  .baga-icon-container .baga-icon {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center; }
```

Comment:
- [OPTIONAL ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```
.d-flex {
  display: flex;
  justify-content: center;
}
```
- HTML file
```
<div class="d-flex">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉